libwget/utils.c

```diff
@@ -62,6 +63,37 @@
  * They may be useful to other developers that is why they are exported.
  */
 
+static const char * _wget_ssl_default_path(BOOL bundleNotDir)

```

Comment:
Please rename to `ssl_default_path`, no need for name-spacing with "wget_", and a leading underscore is reserved in C.